### ISSUE TITLE
Combo: Please add ability to separate what notes each SV Type setting applies to

### ISSUE THREAD

**Issue by Emik03** [kloi34/amoguSV#11]
Currently the Combo option forces both SVs to apply to the same notes. If it were possible to individually add one set of notes and another, it would make blending of multiple SV types much easier. An example would be selecting all notes for SV Type 1 and applying a stutter effect, then selecting only key percussion/melody for SV Type 2 and using a slow exponential speed up. Combining both this way would allow making dynamic stuttering far easier to implement.

**Comment by Emik03** [comment]
I meant something like the Splitscroll feature, where you can capture notes onto a layer. I meant that I wanted the ability to choose which notes are affected *per* SV Type, such that you can combine something applied to every note with something applied over an entire section. In my case I want to apply Exponential on each note so that every note jumps to the next immediately, but I want the spacing between each note to be more dynamic, increasing or decreasing over time, where the second Exponential will be across multiple notes, and I want those mixed together.

**Comment by kloi34** [comment]
For amoguSV v6.3 (which was released a few minutes ago), I added a new tool: Edit SVs > Dynamic Scale. You'd first have to place your SVs, then use this tool to change note spacing across multiple notes. This isn't what you exactly requested (since it's not in the combo menu and takes a separate extra step), but I hope this tool lets you accomplish what you want. If I somehow misunderstood your request again, let me know.
